packages/frontend/app/src/session/Session.tsx

```diff
@@ -30,6 +31,14 @@ export const Session = ({
   presenter,
   getSessions
 }: SessionProps) => {
+
+  const [interest, setInterest] = useState(sessionStorage.checkInterest(id));
```

Comment:
Since actually there's a global sessionStorage, would it be better to find another name?
sessionLocalStorage for example